web/forms.py

```
@@ -1174,13 +1176,26 @@
     def __init__(self, *args, **kwargs):
         user = kwargs.pop("user", None)
         super().__init__(*args, **kwargs)
-
-        # If user is authenticated, remove name, email and captcha fields
+        # If the user is authenticated, remove name, email, and captcha fields.
         if user and user.is_authenticated:
             del self.fields["name"]
             del self.fields["email"]
             del self.fields["captcha"]
 
+    def clean_message(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `clean_message`

(ANN201)